wiki/releasing-versions.md

```diff
@@ -59,7 +59,6 @@ This provides a walkthrough of the patching & backport release process; examples
 * Once everything looks correct, it's time to release; the `yarn release` script only works when releasing from `master`, so we'll run [a subset of those steps](https://github.com/elastic/eui/blob/06fc9a6880766168aec1a622873e7f6fe1b3d42b/scripts/release.js#L34-L57) manually
   * Run the unit tests again - `npm test`
   * Create the release builds - `npm run build`
-  * Update the I18n tokens - `npm run update-token-changelog -- patch`
```

Comment:
Do we have an alternative way of pulling these i18n tokens?

Reply:
I believe i18n tokens are pulled with the `extract-i18n-strings` script. I think this script might have generated some sort of changelog, but it's been broken for a while, so not something that we've even used

Reply:
This script generated the `i18ntokens_changelog.json` file